src/app/[countryCode]/page.tsx

```diff
@@ -0,0 +1,97 @@
+import Image from 'next/image';
+import styles from './page.module.scss';
+import Link from 'next/link';
+import BackButton from '@/components/BackButton';
+import { notFound } from 'next/navigation';
+import renderKeyValuePairs from '@/helperFunctions/renderKeyValuePairs';
+import commafy from '@/helperFunctions/commafy';
+
+async function getData(countryCode: string) {
```

Comment:
What do you think if we create a file for fetching the data? e.g., a folder for fetching data

Reply:
That is a cleaner way to handle fetch functions. Added a separate fetchUtils folder to store fetch functions.